Bio/AlignIO/MafIO.py

```diff
@@ -728,15 +730,60 @@ def get_spliced(self, starts, ends, strand=1):
         # finally, build a MultipleSeqAlignment object for our final sequences
         result_multiseq = []
 
-        for seqid, seq in subseq.items():
-            seq = Seq(seq)
+        # previously, calling get_spliced(starts,ends,-1) with len(starts) > 1 and len(ends) > 1
+        # would return the exons in the wrong order, see issue #1308
+        # basically, the whole sequence would be reverse complemented; instead
+        # each exon has to be reverse complemented, and then concatenated to the resulting sequence
+
+        if strand != ref_first_strand:
+
+            # find how many bases are in each exon
+            offsets = [end - start for start, end in zip(starts, ends)]
+
+            # contains positions where to split
+            positions = [0]
+
+            # this is reference sequence
+            ref_seq = subseq[self._target_seqname]
+
+            # find the positions where to split
+            index = 0
+            for offset in offsets:
+                count_bases = 0
+                for element in ref_seq:
+                    if element == 'A' or element == 'C' or element == 'G' or element == 'T':
+                        count_bases += 1
+                    index += 1
+                    if count_bases == offset:
+                        positions.append(index)
+
+            for seqid, seq in subseq.items():
+                temp_seq = ""
+
+                # split the sequence into exons
+                exons = [seq[i:j] for i, j in zip(positions, positions[1:])]
+
+                for exon in exons:
+                    # convert exon (string) to seq in order to reverse and complement it
+                    seq = Seq(exon).reverse_complement()
```

Comment:
If you are working with strings, just use the ``reverse_complement()`` function instead or converting to ``Seq`` object and back again.

i.e. ``from Bio.Seq import reverse_complement as _rc`` or similar.

Reply:
Are you sure ``subseq`` isn't already ``Seq`` objects? If it is, you are discarding the alphabet here.

Reply:
1) Done!
2) Yes, I tried `print(type(seq))` and got `<type 'str'>`